tests/bluetooth/test_emulation.py

```diff
@@ -95,6 +104,8 @@ async def test_simulate_create_adapter_twice(websocket, context_id, state_1,
             }
         })
 
+    await disable_simulation(websocket, context_id)
```

Comment:
It looks like `disable_simulation` is added for cleaning up the bluetooth state. If so, it should be moved to the beginning of the test, so that failing test would not influence another tests.

Reply:
Done. Added a fixture with `autouse=true` to be used in every test.